politeiad/api/v1/v1.go

```diff
@@ -264,3 +289,18 @@ type InventoryReply struct {
 	Vetted   []ProposalRecord `json:"vetted"`   // Last N vetted proposals
 	Branches []ProposalRecord `json:"branches"` // Last N branches (censored, new etc)
 }
+
+// UserErrorReply returns details about an error that occurred while trying to
+// execute a command due to bad input from the client.
+type UserErrorReply struct {
+	ErrorCode    StatusT     `json:"errorcode"`              // Numeric error code
+	ErrorMessage string      `json:"errormessage,omitempty"` // Human readable error string
```

Comment:
You should drop the ErrorMessage and put that in the API as a map.  That way localization later doesn't turn into a turd.  The entire point of error codes is to have easy checks.  When presenting this to the user you pull the human-readable part out of the map.

Reply:
Dropped ErrorMessage.